### ISSUE TITLE
Performance of BitcoinTransaction hash lookups

### ISSUE THREAD

**Issue by fbruzzaniti** [straumat/blockchain2graph#596]
Just wondering if it's worth while adding an index on BitcoinTransaction txid
I.e. CREATE INDEX ON :BitcoinTransaction(hash)

**Comment by straumat** [comment]
Indeed, seems it's no more useful as I'm searching with txid and n. I will remove it in the next release (when neo4j 4 will be supported by spring) and I will run the benchmark
